php/Database.php

```diff
@@ -184,23 +184,25 @@ public function MigrateDatabase() {
                 $escapedIP = $this->EscapeString($ip);
                 $escapedUserAgent = $this->EscapeString($userAgent);
                 $escapedLog = $this->EscapeString("Running $newDatabaseVersionMigrationFile to bring DATABASE_VERSION up to $newDatabaseVersion");
-                $sql = "
-                    INSERT INTO admin_log
-                    (log_id, log_datetime, log_ip, log_user_agent, log_admin_username_override, log_admin_user_id, log_subject_user_id, log_type, log_content)
-                    VALUES
-                    (
-                        null,
-                        Now(),
-                        '$escapedIP',
-                        '$escapedUserAgent',
-                        'AUTOMATIC',
-                        NULL,
-                        NULL,
-                        'DB_MIGRATION',
-                        '$escapedLog'
-                    );";
-                $this->Execute($sql) or die("Migration failed to log to admin log, please notify site admin");
-                $sql = "";
+                if ($newDatabaseVersion > 14) { // older DB versions have an outdated admin_log table
+                    $sql = "
+                        INSERT INTO admin_log
+                        (log_id, log_datetime, log_ip, log_user_agent, log_admin_username_override, log_admin_user_id, log_subject_user_id, log_type, log_content)
+                        VALUES
+                        (
+                            null,
+                            Now(),
+                            '$escapedIP',
+                            '$escapedUserAgent',
+                            'AUTOMATIC',
+                            NULL,
+                            NULL,
+                            'DB_MIGRATION',
+                            '$escapedLog'
+                        );";
+                    $this->Execute($sql) or die("Migration failed to log to admin log, please notify site admin");
+                    $sql = "";
+                }
```

Comment:
This solution should solve the issue that prompted the "Aug 2020" note in readme.md part 4: https://github.com/OneHourGameJam/OneHourGameJam#4-install-website - please remove that note too, thanks